src/main/java/uk/gov/hmcts/ccd/domain/service/createevent/DefaultCreateEventOperation.java

```diff
@@ -75,6 +76,7 @@
     private final SecurityClassificationService securityClassificationService;
     private final ValidateCaseFieldsOperation validateCaseFieldsOperation;
     private final UserAuthorisation userAuthorisation;
+    private final TransactionHelper transactionHelper;
 
     @Inject
     public DefaultCreateEventOperation(@Qualifier(CachedUserRepository.QUALIFIER) final UserRepository userRepository,
```

Comment:
do we have a similar issue on case creation? i.e. `DefaultCreateCaseOperation`

Reply:
good point. let me check that code as well.

Reply:
Yes :) shall i apply same fix to that flow?